Fix scripts/updating incident records and disables associated workflows  based on specific criteria using two functions /script.js

```diff
@@ -0,0 +1,23 @@
+updateWorkNotes();
+updateResolve();
+function updateWorkNotes() {
+	var gr = new GlideRecord('incident');
+	gr.addEncodedQuery('priority=1^state=2^category=software');
+	gr.query();
+	while(gr.next()) {
```

Comment:
When you update multiple records matching a criteria with the same values, using a while loop is quite inefficient. Instead, the [.updateMultiple() method](https://developer.servicenow.com/dev.do#!/reference/api/vancouver/server_legacy/c_GlideRecordAPI#r_GlideRecord-updateMultiple?navFilter=updatemultiple) of GlideRecord should be used.